api/V1/Visits.ts

```diff
@@ -358,30 +367,53 @@ class Visit {
     this.addRecording(rec, userID);
   }
 
-  mostCommonTag(): string | null {
+  mostCommonTag(): [boolean | null, string | null] {
     // from all events in a visit, get the tag with the highest occurence that
-    // isnt unidentified
+    // isnt unidentified, preferring human tags over ai
+    // returns [boolean desciribing if human tag, the tag]
     const tagCount = this.tagCount;
     const sortedKeys = Object.keys(tagCount).sort(function (a, b) {
-      if (a == unidentified) {
+      let hyphen = a.indexOf("-");
```

Comment:
This is all quite mysterious reading it.   Maybe it would be simpler if you define a type for the key, so you can use a.type and a.what.    Else make a function split to do the splitting.   You repeat the splitting three times in this one function.  

May just be easier to have a list of human tags and a list of ai tags run through the whole procedure.

Reply:
Good point have update to save the track tag